sql/moz-fx-data-shared-prod/firefox_desktop_derived/pageload_nightly_v1/metadata.yaml

```diff
@@ -0,0 +1,23 @@
+friendly_name: Pageload Nightly
+description: |-
+  Contains pings from nightly channel only from pageload_v1
+owners:
+- wichan@mozilla.com
+- acreskey@mozilla.com
+- dpalmeiro@mozilla.com
+labels:
+  incremental: true
+scheduling:
+  dag_name: bqetl_pageload_v1
+  task_name: firefox_desktop_pageload_nightly__v1
```

Comment:
Is there a specific reason why we create separate tables in the `_derived` dataset vs having a table unioning all of them and clustering on channel?

Reply:
I believe it's because of the bytes scanned limit we have in redash (20 TB?) which uses the dry run estimate which usually doesn't take clustering into account